src/service/video-manager-impl.js

```diff
@@ -819,14 +819,22 @@ class VideoEntry {
 
     /** @param {boolean} isPlaying */
     const toggleAnimation = (isPlaying) => {
-      video.mutateElement(() => {
-        animation.classList.toggle('amp-video-eq-play', isPlaying);
-      });
+      video.mutateElement(
+        () => {
+          animation.classList.toggle('amp-video-eq-play', isPlaying);
+        },
+        undefined,
+        /* skipRemeasure */ true
+      );
     };
 
-    video.mutateElement(() => {
-      element.appendChild(animation);
-    });
+    video.mutateElement(
+      () => {
```

Comment:
Does the caller do anything with the void response? If not, it might be safe to convert.

`() => element.appendChild(animation)`

Additionally having to pass an `undefined` second value is a bit of a bummer.

Reply:
This should be fine to turn into into immediate return.